framework/src/Volo.Abp.Ddd.Domain/Volo/Abp/Domain/Entities/Caching/IEntityCache.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Threading.Tasks;
+using Volo.Abp.Caching;
+
+namespace Volo.Abp.Domain.Entities.Caching
+{
+    public interface IEntityCache<TCacheItem, TKey>
+        where TCacheItem : class
+    {
+        IDistributedCache<TCacheItem> InternalCache { get; }
+
+        TCacheItem this[TKey key] { get; }
+
+        string CacheName { get; }
+
+        TCacheItem Get(TKey key);
```

Comment:
Don't add sync version. We don't want to use `AsyncHelper.RunSync`. If the developers wants, they can use `AsyncHelper.RunSync` while using the `GetAsync`.